Misc/Applications/lib/foldGrammars/Testing.pm

```diff
@@ -238,10 +238,15 @@ sub addRandomParameters {
 
 sub evaluateTest {
 	my ($testname, $truth, $TMPDIR, $testIndex, $refList_failedTests) = @_;
-	
+
 	my $status = 'failed';
 	if (-e "Truth/".$truth) {
-		my $diffResult = Utils::execute("diff -I \"^#CMD:\" Truth/$truth $TMPDIR/$truth"); chomp $diffResult;
+		# Replace concrete prefix file paths from executed test file with generic ROOTDIR and GAPDIR strings.
```

Comment:
Not sure but shouldn't it be "ROOTDIR and *B*_GAPDIR ?

Reply:
good catch